types/luxon/src/datetime.d.ts

```diff
@@ -961,7 +1027,7 @@ export class DateTime {
      *                                 //=>   { type: 'year', value: '1982' }
      *                                 //=> ]
      */
-    toLocaleParts(opts: DateTimeFormatOptions): Intl.DateTimeFormat[];
+    toLocaleParts(opts?: DateTimeFormatPreset | DateTimeFormatOptions): ToLocalePartsOutput;
```

Comment:
So here there was just a wrong return, why not amend to `Intl.DateTimeFormatPart[]`?

Reply:
Absolutely, I didn't thoroughly check Intl for an equivalent. Amending.